go.mod

```diff
@@ -43,7 +43,7 @@ require (
 	golang.org/x/tools v0.9.3
 	gopkg.in/yaml.v2 v2.4.0
 	k8s.io/api v0.27.0
-	k8s.io/apimachinery v0.27.3
+	k8s.io/apimachinery v0.28.0
```

Comment:
@naftalysh Why are you changing version of `apimachinery` package? It looks like an accidental inclusion of the `go.mod` in the PR. Do you have any specific reason to update that version?

Reply:
If you don't need that particular version, please remove the change from the PR. thx

Reply:
While trying to load test my updated project I received warning messages stating I should run go mod vendor 
I've done that and also go mod tidy and that I think updated the go.mod which is a good practice I think

Reply:
@pmacik restored previous version